job.c

```diff
@@ -176,24 +176,24 @@ job_pri_less(void *ja, void *jb)
 int
 job_delay_less(void *ja, void *jb)
 {
-    job a = ja;
-    job b = jb;
+    Job *a = ja;
+    Job *b = jb;
     if (a->r.deadline_at < b->r.deadline_at) return 1;
     if (a->r.deadline_at > b->r.deadline_at) return 0;
     return a->r.id < b->r.id;
 }
 
-job
-job_copy(job j)
+Job *
+job_copy(Job *j)
 {
-    job n;
+    Job *n;
 
     if (!j) return NULL;
 
-    n = malloc(sizeof(struct job) + j->r.body_size);
-    if (!n) return twarnx("OOM"), (job) 0;
+    n = malloc(sizeof(struct Job) + j->r.body_size);
+    if (!n) return twarnx("OOM"), (Job *) 0;
 
-    memcpy(n, j, sizeof(struct job) + j->r.body_size);
+    memcpy(n, j, sizeof(struct Job) + j->r.body_size);
```

Comment:
Optionally I think these could say `sizeof(Job)` now if you wanted.

Reply:
_Job's done._